modules/terraform/tf-options.nix

```
@@ -298,9 +298,24 @@
           # Git commit encrypted state
           echo "  Committing encrypted state       ..."
           git -C "$WORKTREE" add "$WORKTREE/$ENC_STATE_PATH" &>> "$WORKLOG"
-          commitPrompt
-          git -C "$WORKTREE" commit --no-verify -m "$(echo -e "$(printf '%s' "''${MSG[@]}")")" &>> "$WORKLOG"
-          git -C "$WORKTREE" push -u "$REMOTE" "$TF_BRANCH" &>> "$WORKLOG"
+          MESSAGE="$(echo -e "$(printf '%s' "''${MSG[@]}")")"
+          if [[ -v CI && $CI == 'true' ]] && type gh &>/dev/null; then
+            SHA="$(git rev-parse "$TF_BRANCH:$ENC_STATE_PATH")"
```

```suggestion
          if [[ -v CI && "$CI" == true ]] && type gh &>/dev/null; then
            SHA=$(git rev-parse "$TF_BRANCH:$ENC_STATE_PATH")
```

- `$CI` could possibly have whitespace (granted probably not in this script)
- quoting `true` is not necessary
- quoting is not necessary when assigning `$()` to a variable